pkg/clusteragent/autoscaling/workload/model/pod_autoscaler_string.go

```diff
@@ -0,0 +1,417 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+package model
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	datadoghqcommon "github.com/DataDog/datadog-operator/api/datadoghq/common"
+	"github.com/DataDog/datadog-operator/api/datadoghq/v1alpha2"
+)
+
+func (p *PodAutoscalerInternal) String(verbose bool) string {
```

Comment:
I know it's the pattern we have in WLM, but given the smaller scope, I am wondering if it's worth having an auto printer instead? Like https://github.com/sanity-io/litter

Reply:
Started playing around with it, having some trouble getting it formatted nicely (e.g. for timestamps), will keep looking into it!